pandas/tests/test_downstream.py

```diff
@@ -47,6 +47,18 @@ def test_xarray(df):
     assert df.to_xarray() is not None
 
 
+def test_xarray_cftimeindex_nearest():
+    # https://github.com/pydata/xarray/issues/3751
+    cftime = import_module("cftime")  # noqa
+    xarray = import_module("xarray")  # noqa
```

Comment:
why do these need noqa?

Reply:
Oops, indeed they don't; I mindlessly copied from the test above.